units/en/unit8/introduction-sf.mdx

```diff
@@ -0,0 +1,13 @@
+# Introduction to PPO with Sample-Factory
+
+<img src="https://huggingface.co/datasets/huggingface-deep-rl-course/course-images/resolve/main/en/unit9/thumbnail2.png" alt="thumbnail"/>
+
+In this second part of Unit 8, we'll get deeper into PPO optimization by using [Sample-Factory](https://samplefactory.dev/), an **asynchronous implementation of the PPO algorithm**, to train our agent playing [vizdoom](https://vizdoom.cs.put.edu.pl/) (an open source version of Doom).
+
+During the notebook, **you'll train your agent to play Health Gathering level**, where our agent must collect health packs to avoid dying. And after that, you can **train your agent to play more complex versions of the levels, such as Deathmatch**.
```

Comment:
```suggestion
In the notebook, **you'll train your agent to play the Health Gathering level**, where the agent must collect health packs to avoid dying. After that, you can **train your agent to play more complex levels, such as Deathmatch**.
```